apps/web/src/routes/[ownerSlug]/[projectSlug]/reviews/[branchId]/commit/[changeId]/+page.svelte

```diff
@@ -1,22 +1,64 @@
 <script lang="ts">
 	import ChatComponent from '$lib/components/ChatComponent.svelte';
+	import { BranchService } from '@gitbutler/shared/branches/branchService';
+	import { getBranchReview } from '@gitbutler/shared/branches/branchesPreview.svelte';
+	import { PatchService } from '@gitbutler/shared/branches/patchService';
+	import { getPatch } from '@gitbutler/shared/branches/patchesPreview.svelte';
+	import { getContext } from '@gitbutler/shared/context';
+	import Loading from '@gitbutler/shared/network/Loading.svelte';
+	import { isFound } from '@gitbutler/shared/network/loadable';
+	import { lookupProject } from '@gitbutler/shared/organizations/repositoryIdLookupPreview.svelte';
+	import { RepositoryIdLookupService } from '@gitbutler/shared/organizations/repositoryIdLookupService';
+	import { AppState } from '@gitbutler/shared/redux/store.svelte';
 	import type { ProjectReviewCommitParameters } from '$lib/project/types';
 
 	interface Props {
 		data: ProjectReviewCommitParameters;
 	}
 
 	let { data }: Props = $props();
-</script>
 
-<h2>Review page: {data.ownerSlug}/{data.projectSlug} {data.branchId}/{data.changeId}</h2>
+	const repositoryIdLookupService = getContext(RepositoryIdLookupService);
+	const branchService = getContext(BranchService);
+	const patchService = getContext(PatchService);
+	const appState = getContext(AppState);
+
+	const repositoryId = $derived(
+		lookupProject(appState, repositoryIdLookupService, data.ownerSlug, data.projectSlug)
+	);
+
+	const branch = $derived(
+		isFound(repositoryId.current)
+			? getBranchReview(appState, branchService, repositoryId.current.value, data.branchId)
+			: undefined
+	);
+
+	const branchUUID = $derived(isFound(branch?.current) ? branch.current.value.uuid : undefined);
+
+	const change = $derived(
+		branchUUID !== undefined
+			? getPatch(appState, patchService, branchUUID, data.changeId)
+			: undefined
+	);
+</script>
```

Comment:
Minor thing, but I talking to mattias `branchUuid` would be preferred over `branchUUID`